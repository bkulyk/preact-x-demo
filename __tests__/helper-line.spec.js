import React from 'react';
import HelperLine from '@bitchin/helper-line';
import { shallow } from './enzyme';
import { it, expect } from './jest';

it('should render correctly', () => {
  expect(shallow((<HelperLine className="custom">children</HelperLine>)))
    .toMatchSnapshot();
});
