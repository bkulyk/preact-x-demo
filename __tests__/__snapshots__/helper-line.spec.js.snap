// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="mdc-text-field-helper-line"
>
  <p
    className="mdc-text-field-helper-text mdc-text-field-helper-text--persistent mdc-text-field-helper-text--validation-msg custom"
  >
    children
  </p>
</div>
`;
